tests/unit/admin.test.tsx

```diff
@@ -1,12 +1,48 @@
 import '@testing-library/jest-dom'
 import { render, screen } from '@testing-library/react'
 import Page from '../../app/admin/page';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+
+// Mock next/link component
+jest.mock('next/link', () => {
+    return ({ children, href }) => {
```

Comment:
The linter is warning us of undeclared types. 
You can add the following to fix the lint warnings:
`return ({ children, href }: {children: React.ReactNode, href: string}) => {`

Reply:
Thank you, this helped a lot!